src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -315,7 +314,10 @@
     {% set form_method = "POST" %}
   {%- endif -%}
   <form name="{{ name }}"
-        method="{{ form_method|lower }}" action="{{ action }}"{% for attrname, attrvalue in attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}{% if multipart %} enctype="multipart/form-data"{% endif %}>
+        method="{{ form_method|lower }}" 
```

Comment:
Same comment as bootstrap4 layout, this file will stop being used and probably will be removed.